lib/msal-core/test/telemetry/TelemetryUtils.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from "chai";
+import { scrubTenantFromUri, hashPersonalIdentifier } from "../../src/telemetry/TelemetryUtils";
+
+describe("TelemetryUtils", () => {
+    it("TelemtryUtils.scrubTenantFromUri works on regular uri", () => {
+        const uri = scrubTenantFromUri("https://login.microsoftonline.com/abc-123/banana/orange");
+        expect(uri).to.eq("https://login.microsoftonline.com/abc-123/<tenant>/orange");
+    });
+    it("TelemtryUtils.scrubTenantFromUri returns null on untrusted uri", () => {
+        const uri = scrubTenantFromUri("https://pizza.burger.com/abc-123/banana/orange");
+        expect(uri).to.be.null;
+    });
+    it("TelemtryUtils.scrubTenantFromUri doesnt change when it doesnt contain enough path params", () => {
+        const uri = scrubTenantFromUri("https://login.microsoftonline.com/abc-123/");
+        expect(uri).to.eq("https://login.microsoftonline.com/abc-123");
+    });
+    it("TelemtryUtils.scrubTenantFromUri works for b2c", () => {
+        const uri = scrubTenantFromUri("https://login.microsoftonline.com/abc-123/tfp/banana");
+        expect(uri).to.eq("https://login.microsoftonline.com/abc-123/tfp/<tenant>");
+    });
+    it("TelemtryUtils.scrubTenantFromUri works for b2c", () => {
+        const uri = scrubTenantFromUri("https://login.microsoftonline.com/abc-123/tfp/banana");
+        expect(uri).to.eq("https://login.microsoftonline.com/abc-123/tfp/<tenant>");
+    });
```

Comment:
I think this is the same test twice in a row?

Reply:
I'm guessing you meant to create a second test without the b2c prefix?

Reply:
good catch, not sure what I meant there!